From a670bb7d9229ab9b00d12dfe8e2801ebbdfe6ede Mon Sep 17 00:00:00 2001
From: Tanmay Jagdale <tanmayj@broadcom.com>
Date: Mon, 15 Dec 2014 16:30:41 +0530
Subject: pic: added compatible property to PIC node

Added a 'compatible' property to the PIC node in all the
*-linux-kernel.dts files.
[Based on SDK 3.2]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
Signed-off-by: Nam Ninh <nam.ninh@windriver.com>

diff --git a/arch/mips/boot/dts/xlp2xx-ep-linux-kernel.dts b/arch/mips/boot/dts/xlp2xx-ep-linux-kernel.dts
index afeddac..d579035 100644
--- a/arch/mips/boot/dts/xlp2xx-ep-linux-kernel.dts
+++ b/arch/mips/boot/dts/xlp2xx-ep-linux-kernel.dts
@@ -80,6 +80,7 @@
                         };
 		};
 		pic: pic@4000 {
+			compatible = "netlogic,xlp-pic";
 			interrupt-controller;
 			#address-cells = <0>;
 			#interrupt-cells = <1>;
diff --git a/arch/mips/boot/dts/xlp2xx-linux-kernel.dts b/arch/mips/boot/dts/xlp2xx-linux-kernel.dts
index b9b791d..5baa5d9 100644
--- a/arch/mips/boot/dts/xlp2xx-linux-kernel.dts
+++ b/arch/mips/boot/dts/xlp2xx-linux-kernel.dts
@@ -122,6 +122,7 @@
                         };
 		};
 		pic: pic@4000 {
+			compatible = "netlogic,xlp-pic";
 			interrupt-controller;
 			#address-cells = <0>;
 			#interrupt-cells = <1>;
diff --git a/arch/mips/boot/dts/xlp2xx-pcie-linux-kernel.dts b/arch/mips/boot/dts/xlp2xx-pcie-linux-kernel.dts
index fb13ff3..36be059 100644
--- a/arch/mips/boot/dts/xlp2xx-pcie-linux-kernel.dts
+++ b/arch/mips/boot/dts/xlp2xx-pcie-linux-kernel.dts
@@ -80,6 +80,7 @@
                         };
 		};
 		pic: pic@4000 {
+			compatible = "netlogic,xlp-pic";
 			interrupt-controller;
 			#address-cells = <0>;
 			#interrupt-cells = <1>;
diff --git a/arch/mips/boot/dts/xlp2xx-xmc-linux-kernel.dts b/arch/mips/boot/dts/xlp2xx-xmc-linux-kernel.dts
index 355230f..6455bfb 100644
--- a/arch/mips/boot/dts/xlp2xx-xmc-linux-kernel.dts
+++ b/arch/mips/boot/dts/xlp2xx-xmc-linux-kernel.dts
@@ -69,6 +69,7 @@
                         };
 		};
 		pic: pic@4000 {
+			compatible = "netlogic,xlp-pic";
 			interrupt-controller;
 			#address-cells = <0>;
 			#interrupt-cells = <1>;
diff --git a/arch/mips/boot/dts/xlp3xx-linux-kernel.dts b/arch/mips/boot/dts/xlp3xx-linux-kernel.dts
index c1f763a..99159ab 100644
--- a/arch/mips/boot/dts/xlp3xx-linux-kernel.dts
+++ b/arch/mips/boot/dts/xlp3xx-linux-kernel.dts
@@ -74,6 +74,7 @@
                         };
 		};
 		pic: pic@4000 {
+			compatible = "netlogic,xlp-pic";
 			interrupt-controller;
 			#address-cells = <0>;
 			#interrupt-cells = <1>;
diff --git a/arch/mips/boot/dts/xlp3xx-xmc-linux-kernel.dts b/arch/mips/boot/dts/xlp3xx-xmc-linux-kernel.dts
index 46e7fdf..f966f22 100644
--- a/arch/mips/boot/dts/xlp3xx-xmc-linux-kernel.dts
+++ b/arch/mips/boot/dts/xlp3xx-xmc-linux-kernel.dts
@@ -75,6 +75,7 @@
 
 		};
 		pic: pic@4000 {
+			compatible = "netlogic,xlp-pic";
 			interrupt-controller;
 			#address-cells = <0>;
 			#interrupt-cells = <1>;
diff --git a/arch/mips/boot/dts/xlp4xx-linux-kernel.dts b/arch/mips/boot/dts/xlp4xx-linux-kernel.dts
index 8363451..bc044e9 100644
--- a/arch/mips/boot/dts/xlp4xx-linux-kernel.dts
+++ b/arch/mips/boot/dts/xlp4xx-linux-kernel.dts
@@ -74,6 +74,7 @@
                         };
 		};
 		pic: pic@4000 {
+			compatible = "netlogic,xlp-pic";
 			interrupt-controller;
 			#address-cells = <0>;
 			#interrupt-cells = <1>;
diff --git a/arch/mips/boot/dts/xlp5xx-linux-kernel.dts b/arch/mips/boot/dts/xlp5xx-linux-kernel.dts
index 90bbf37..76c5d20 100644
--- a/arch/mips/boot/dts/xlp5xx-linux-kernel.dts
+++ b/arch/mips/boot/dts/xlp5xx-linux-kernel.dts
@@ -84,6 +84,7 @@
 			interrupts = <18>;
 		};
 		pic: pic@11000 {
+			compatible = "netlogic,xlp-pic";
 			interrupt-controller;
 			#address-cells = <0>;
 			#interrupt-cells = <1>;
diff --git a/arch/mips/boot/dts/xlp5xx-pcie-linux-kernel.dts b/arch/mips/boot/dts/xlp5xx-pcie-linux-kernel.dts
index 48df268..568f7cd 100644
--- a/arch/mips/boot/dts/xlp5xx-pcie-linux-kernel.dts
+++ b/arch/mips/boot/dts/xlp5xx-pcie-linux-kernel.dts
@@ -43,6 +43,7 @@
 			interrupts = <18>;
 		};
 		pic: pic@11000 {
+			compatible = "netlogic,xlp-pic";
 			interrupt-controller;
 			#address-cells = <0>;
 			#interrupt-cells = <1>;
diff --git a/arch/mips/boot/dts/xlp8xx-linux-kernel.dts b/arch/mips/boot/dts/xlp8xx-linux-kernel.dts
index be096da..4e91261 100644
--- a/arch/mips/boot/dts/xlp8xx-linux-kernel.dts
+++ b/arch/mips/boot/dts/xlp8xx-linux-kernel.dts
@@ -121,6 +121,7 @@
 			   };
 		};
 		pic: pic@4000 {
+			compatible = "netlogic,xlp-pic";
 			interrupt-controller;
 			#address-cells = <0>;
 			#interrupt-cells = <1>;
diff --git a/arch/mips/boot/dts/xlp9xx-linux-kernel.dts b/arch/mips/boot/dts/xlp9xx-linux-kernel.dts
index 3e91bb5..2593b21 100644
--- a/arch/mips/boot/dts/xlp9xx-linux-kernel.dts
+++ b/arch/mips/boot/dts/xlp9xx-linux-kernel.dts
@@ -85,6 +85,7 @@
 			interrupts = <18>;
 		};
 		pic: pic@11000 {
+			compatible = "netlogic,xlp-pic";
 			interrupt-controller;
 			#address-cells = <0>;
 			#interrupt-cells = <1>;
-- 
1.7.1

